app/controllers/concerns/course/forum_topics_concern.rb

```diff
@@ -0,0 +1,35 @@
+module Course::ForumTopicsConcern
```

Comment:
We normally group related things into a concern,  so can you make your concern more meaningful? ( i.e not just put all private methods here. )